tests/src/test/java/alluxio/shell/command/CopyFromLocalCommandTest.java

```diff
@@ -358,6 +362,21 @@ public void copyFromLocal() throws IOException, AlluxioException {
   }
 
   @Test
+  public void copyFromLocalMustCacheThenCacheThrough() throws Exception {
+    File file = mTestFolder.newFile();
+    try (Closeable c = new ConfigurationRule(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT,
+        WriteType.MUST_CACHE.toString()).toResource()) {
+      Assert.assertEquals(0, mFsShell.run("copyFromLocal", file.getAbsolutePath(), "/"));
+    }
+    try (Closeable c = new ConfigurationRule(PropertyKey.USER_FILE_WRITE_TYPE_DEFAULT,
+        WriteType.CACHE_THROUGH.toString()).toResource()) {
+      mOutput.reset();
+      mFsShell.run("copyFromLocal", file.getAbsolutePath(), "/");
+    }
+    Assert.assertThat(mOutput.toString(), CoreMatchers.containsString("already exists"));
```

Comment:
Is there a more exact way to do the matching? `already exists` seems a bit arbitrary.